plutus-metatheory/src/Raw.lagda

```diff
@@ -107,6 +108,9 @@ decBuiltin sha2-256 sha2-256 = true
 decBuiltin sha3-256 sha3-256 = true
 decBuiltin verifySignature verifySignature = true
 decBuiltin equalsByteString equalsByteString = true
+decBuiltin chatToString charToString = true
+decBuiltin append append = true
+decBuiltin trace trace = true
```

Comment:
@jmchapman I only accidentally noticed that function after adding the new builtins. Also decidable equality would help. In particular here we can define an injection into `Nat` and derive the equality check function via that injection, so no need to handle O(n^2) cases.

Are there other function that I might have failed to update?